.github/workflows/docker.yml

```diff
@@ -0,0 +1,54 @@
+name: Build Docker Images
+
+on:
+  workflow_dispatch:
+  # push:
+  #   branches:
+  #     - main
+
+env:
+  REGISTRY: ghcr.io
+  RUNTIME_BASE_IMAGE: debian:latest
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+      attestations: write
+      id-token: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push Docker image
+        id: push
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          file: ./Dockerfile
+          cache-from: commontools/base
+          push: true
+          provenance: true
+          tags:
+            ${{ env.REGISTRY }}/${{ github.repository_owner }}/base:${{ github.sha }}
+            ${{ env.REGISTRY }}/${{ github.repository_owner }}/base:latest
+          build-args: |
+            BASE_IMAGE=${{ env.RUNTIME_BASE_IMAGE }}
+
+      - name: Generate artifact attestation
+        uses: actions/attest-build-provenance@v1
+        with:
+          subject-name: ${{ env.REGISTRY }}//${{ github.repository_owner }}/base
```

Comment:
Is there an extra `/` here, or is that intentional?

Reply:
ah nice catch
```suggestion
          subject-name: ${{ env.REGISTRY }}/${{ github.repository_owner }}/base
```